### ISSUE TITLE
[Optimization] Make sure to tag every service with an "Environment"

### ISSUE THREAD

**Issue by davidd8** [filecoin-project/storetheindex#1043]
Currently over 50% of the services running do not have an associated Environment tag. Let's tag appropriately to better analyze usage.

**Comment by masih** [comment]
Are these indexing related services?

**Comment by davidd8** [comment]
No, this is based on all services running in the STI account, which I assume to be predominantly indexer-related. But by tagging we can be sure.

**Comment by masih** [comment]
All indexer related services are tagged using common tags specified in terraform. 

If untagged services are not indexing related I suggest we move this issue to the relevant repo, e.g. autoretrieve-deploy or boost.
